ui/css/main.css

```diff
@@ -2000,6 +2000,10 @@ body  { margin: 0;}
   text-align: center !important;
 }
 
+.info .button.are-you-sure-button {
```

Comment:
This isn't critical but in the future we should keep css selectors flat (i.e., just ``.are-you-sure-button``) because deep selectors impact performance.

Reply:
I started doing just `.are-you-sure-button` but it was loosing in priority to `.info .button`.